lib/src/base.dart

```diff
@@ -194,15 +194,15 @@ class FlutterWebviewPlugin {
   }
 
   Future<Map<String, String>> getCookies() async {
-    final cookiesString = await evalJavascript('document.cookie');
+    var cookiesString = await evalJavascript('document.cookie');
     final cookies = <String, String>{};
 
-    if (cookiesString?.isNotEmpty == true) {
-      cookiesString.split(';').forEach((String cookie) {
-        final split = cookie.split('=');
-        cookies[split[0]] = split[1];
-      });
-    }
+    cookiesString
+      .replaceAll('"', '')
+      .split('; ')
```

Comment:
Sometimes we want to filter empty elements to avoid map items with empty key and empty value:
`.where((c)` => c.isNotEmpty )`
Example: empty cookie produce { empty: empty }

Reply:
You got it, thanks man!